src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -40,6 +40,61 @@ abstract class Widget_Abstract extends \Tribe\Widget\Widget_Abstract {
 	 */
 	protected $view_slug;
 
+	/**
+	 * @todo update in TEC-3612 & TEC-3613
+	 *
+	 * {@inheritDoc}
+	 * {@inheritDoc}default
+	 */
+	public function setup() {
+		// Add the admin template class for the widget admin form.
+		$this->admin_template = tribe( Admin_Template::class );
+
+		// Setup the View for the frontend.
+		$this->setup_view();
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function form( $instance ) {
+
+		add_filter(
+			"tribe_widget_{$this->get_registration_slug()}_arguments",
+			function ( array $arguments ) use ( $instance ) {
+				return wp_parse_args(
+					$instance,
+					$arguments
+				);
+		} );
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 2
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: b56682f84e342d817e8a7eab5914aa1f</sub>